eclipse-theia/theia
# deps: perform `yarn upgrade`

#### What it does


The pull-request includes the following changes:
- perform `yarn upgrade` to update dependencies to versions that downstream adopters would pull, and fix known security vulnerabilities
- updated version ranges to compatible versions (ex: `msgpackr` (https://github.com/eclipse-theia/theia/pull/12600))
- upgraded `lerna` and performed `lerna repair` to update deprecated configuration options

#### How to test


- confirm that CI successfully passes
- confirm the output of `yarn audit` compared to `master`

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
